platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -1242,433 +720,501 @@ public void start() {
+     */
+    private void startGossip() {
+        gossipNetwork.startGossip();
+    }
+
+    /**
+     * Stop the gossip subsystem. Called before a reconnect is started.
+     */
+    private void stopGossip() {
+        gossipNetwork.stopGossip();
+    }
+
+    /**
+     * If there are multiple platforms running in the same JVM, each will have a unique instance number.
+     *
+     * @return this platform's instance number
+     */
+    @Override
+    public int getInstanceNumber() {
+        return instanceNumber;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public NodeId getSelfId() {
+        return selfId;
+    }
+
+    /**
+     * returns a name for this Platform, which includes the app, the swirld, the member id, and the member name. This is
+     * useful in the Browser window for showing data about each of the running Platform objects.
+     *
+     * @return the name for this Platform
+     */
+    public String getPlatformName() {
+        // this will be the empty string until the Browser calls setInfoMember. Then it will be correct.
+        return platformName;
+    }
+
+    /**
+     * Stores a new system transaction that will be added to an event in the future. Transactions submitted here are not
+     * given priority. Any priority transactions waiting to be included in an event will be included first.
+     *
+     * @param systemTransaction the new system transaction to be included in a future event
+     * @return {@code true} if successful, {@code false} otherwise
+     */
+    public boolean createSystemTransaction(final SystemTransaction systemTransaction) {
+        return createSystemTransaction(systemTransaction, false);
+    }
+
+    /**
+     * Stores a new system transaction that will be added to an event in the future. Transactions submitted here are not
+     * given priority, meaning any priority transaction waiting to be included in an event will be selected first.
+     *
+     * @param systemTransaction the new system transaction to be included in a future event
+     * @return {@code true} if successful, {@code false} otherwise
+     */
+    public boolean createPrioritySystemTransaction(final SystemTransaction systemTransaction) {
+        return createSystemTransaction(systemTransaction, true);
+    }
+
+    private boolean createSystemTransaction(final SystemTransaction systemTransaction, final boolean priority) {
+        if (systemTransaction == null) {
+            return false;
+        }
+
+        return swirldStateManager.submitTransaction(systemTransaction, priority);
+    }
+
+    /**
+     * A container for the initial state.
+     *
+     * @param signedStateFromDisk the initial signed state loaded from disk
+     * @param initialState        the initial {@link State} object. This is a fast copy of the state loaded from disk
+     */
+    private record LoadedState(SignedState signedStateFromDisk, State initialState) {}
```

Comment:
please do not define a record somewhere in the class between methods that do not belong to the record.